src/components/Loader.tsx

```diff
@@ -1,4 +1,6 @@
-const Loader = () => {
+import React from 'react';
+
+const Loader: React.FC = () => {
```

Comment:
you don't have to pass React.FC, even from props
you can also write like this
const Loader = ({className, ...}: Props) => {};

Reply:
I specified the type for the Loader component as a functional component using React.FC. Absolutely, you're correct! The React.FC type is optional, if we use more specific prop type or even use destructuring in the function parameters to extract the props directly. Do I need to change it? Although, using React.FC is not inherently problematic.

Reply:
@rishuxd Nah its alright, just saying we usually don't use FC anymore as we can simple use Props as type in parameters, you don't have to change it just check that replaceAll function

Reply:
I specified the type for the Loader component as a functional component using React.FC. Absolutely, you're correct! The React.FC type is optional, if we use more specific prop type or even use destructuring in the function parameters to extract the props directly. Do I need to change it? Although, using React.FC is not inherently problematic.

Oh okay, Thanks. And I'm sorry, by mistake I deleted the last comment :) added it in case someone read the convo.